src/plots/mapbox/mapbox.js

```diff
@@ -460,18 +462,21 @@ proto.initFx = function(calcData, fullLayout) {
             optsNow._input.zoom = optsNow.zoom = viewNow.zoom;
             optsNow._input.bearing = optsNow.bearing = viewNow.bearing;
             optsNow._input.pitch = optsNow.pitch = viewNow.pitch;
-
             gd.emit('plotly_relayout', self.getViewEditsWithDerived(viewNow));
         }
-        wheeling = false;
+        if(evt.originalEvent && evt.originalEvent.type === 'mouseup') {
```

Comment:
Does this cover the case where one starts dragging on a mapbox subplot and continues into the graph's margins?

Reply:
Yes, this event is triggered even when the drag ends outside the window